posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -17,10 +17,10 @@ Joseph Cass <https://github.com/Joseph-Cass>
 
 link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html[MicroProfile Config 2.0] is a major version release jam-packed with new features to help make configuring your microservices a breeze. Released as part of link:https://download.eclipse.org/microprofile/microprofile-4.0.1/microprofile-spec-4.0.1.html#microprofile4.0[MicroProfile 4.0], Config 2.0 introduces:
 
-- <<Config-Profile, Config Profile>>- a property, called mp.config.profile, which can be used to determine which set of config property values are used
-- <<ConfigValue, ConfigValue>>- an API class which holds a bunch of useful information about a specified config property
-- <<ConfigProperties, @ConfigProperties>>- an annotation which provides a way to retrieve a number of related config property values, with a shared specified prefix, into a CDI bean.
-- <<Property-Expression, Property Expression>>- a way of defining and expanding config property values inside of other config property values using the `${}` syntax.
+- <<Config-Profile, Config Profile>>- A special property, called `mp.config.profile`, can be used to determine which set of config property values are used.
+- <<ConfigValue, ConfigValue>>- An API class which holds a variety of useful information about a specified config property.
+- <<ConfigProperties, @ConfigProperties>>- An annotation which provides a way to retrieve a number of related config property values, with a shared specified prefix, into a CDI bean.
+- <<Property-Expression, Property Expression>>- A way of embedding config property values inside of other config property values using the `${}` syntax.
```

Comment:
Change,
"A way of embedding config property values inside of other config property values using the `${}` syntax."
to,
"A way of embedding config property values inside of other config property values by using the `${}` syntax."